modules/nf-core/trycycler/subsample/tests/main.nf.test

```diff
@@ -0,0 +1,64 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
+// nf-core modules test trycycler/subsample
+nextflow_process {
+
+    name "Test Process TRYCYCLER_SUBSAMPLE"
+    script "../main.nf"
+    process "TRYCYCLER_SUBSAMPLE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "trycycler"
+    tag "trycycler/subsample"
+
+    // TODO nf-core: Change the test name preferably indicating the test-data and file-format used
+    test("candidatus_portiera_aleyrodidarum - default") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/prokaryotes/candidatus_portiera_aleyrodidarum/nanopore/fastq/test.fastq.gz', checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+                //TODO nf-core: Add all required assertions to verify the test output.
+                // See https://nf-co.re/docs/contributing/tutorials/nf-test_assertions for more information and examples.
+            )
+        }
+
+    }
+
+    test("candidatus_portiera_aleyrodidarum - stub") {
+
+        when {
+            process {
+                options "--stub-run"
```

Comment:
I think this should just be --stub, the snapshot files are the same.
```suggestion
                options "--stub"
```

Reply:
It needed to be "-stub-run", fixed in [d9cc745](https://github.com/nf-core/modules/pull/5654/commits/d9cc745b6a04740355262bcfa49fe0c8a11824c6)